src/commands/Management/Helpers/guild-info.ts

```diff
@@ -1,27 +1,42 @@
-import { GuildInfoCMD } from '#lib/commands/guildInfo';
 import { BirthdayyCommand } from '#lib/structures';
 import { PermissionLevels } from '#lib/types/Enums';
+import { OWNERS } from '#root/config';
 import generateConfigList from '#utils/birthday/config';
 import { getFormattedTimestamp } from '#utils/common';
 import { generateDefaultEmbed } from '#utils/embed';
 import { isCustom } from '#utils/env';
 import { getCommandGuilds } from '#utils/functions';
 import { ApplyOptions } from '@sapphire/decorators';
 import { ApplicationCommandRegistry, CommandOptionsRunTypeEnum } from '@sapphire/framework';
+
 @ApplyOptions<BirthdayyCommand.Options>({
+	name: 'guild-info',
 	description: 'Get Infos about a Guild',
 	enabled: !isCustom,
 	permissionLevel: PermissionLevels.Administrator,
-	runIn: CommandOptionsRunTypeEnum.GuildAny
+	preconditions: ['BotOwner'],
```

Comment:
```suggestion

```

Use permissionsLevels

Reply:
Preconditions gives us error messages, when something fails no? What does the permissionLevel do exactly?

Reply:
<img width="1059" alt="Capture d’écran 2024-05-15 à 00 46 14" src="https://github.com/BirthdayyBot/BirthdayyBot/assets/52933626/1aa95b5d-0b04-4287-8a86-87a758b4322d">

As indicated here, it implements the preconditions in such a way that we can always bypass them by being the bot owners